src/test/java/com/snack/news/controller/PicksControllerTest.java

```diff
@@ -0,0 +1,61 @@
+package com.snack.news.controller;
+
+import com.snack.news.exception.advice.ControllerExceptionHandler;
+import com.snack.news.service.PicksService;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.setup.MockMvcBuilders;
+import org.springframework.web.method.HandlerMethod;
+import org.springframework.web.method.annotation.ExceptionHandlerMethodResolver;
+import org.springframework.web.servlet.mvc.method.annotation.ExceptionHandlerExceptionResolver;
+import org.springframework.web.servlet.mvc.method.annotation.ServletInvocableHandlerMethod;
+
+import java.lang.reflect.Method;
+import java.util.Objects;
+
+import static com.snack.news.controller.ApiUrl.Domain.PICKS;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+@ExtendWith(MockitoExtension.class)
+public class PicksControllerTest {
+
+	@InjectMocks
+	private PicksController picksController;
+
+	@Mock
+	private PicksService picksService;
+
+	private static MockMvc mockMvc;
+
+	@BeforeEach
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/manual/snack-news/Snack-BE/pullRequest?prid=4962836)